style.css

```diff
@@ -450,3 +474,290 @@ textarea:invalid:required {
+    font-size: 20px;
+  }
+
+  .flex-connect {
+    margin-left: 8em;
+  }
+
+  .flex-connect h3 {
+    font-size: 16px;
+    font-weight: 500;
+  }
+
+  .foot-connect {
+    margin-left: 6em;
+  }
+
+  .foot-connect ul {
+    max-width: 20%;
+  }
+
+  .grid-container {
+    margin-bottom: 6rem;
+    margin-left: 3rem;
+  }
+
+  .template-wrapper {
+    margin-right: 5rem;
+  }
+
+  .wrapping {
+    display: flex;
+  }
+
+  .mobile-image {
+    display: none;
+  }
+
+  .desk-image {
+    display: block;
+  }
+
+  .desktop-tonic {
+    margin: 24px;
+    padding-left: 25px;
+    padding-right: 50px;
+  }
+
+  .Tonic h2 {
+    font-size: 40px;
+    font-weight: 700;
+  }
+
+  .Tonic-list ul {
+    padding-bottom: 15px;
+  }
+
+  .Tonic-content p {
+    padding-bottom: 10px;
+  }
+
+  .Tonic-link ul {
+    padding-bottom: 20px;
+  }
+
+  .two {
+    order: 1;
+  }
+
+  .canopy {
+    font-weight: 500;
+    font-size: 18px;
+  }
+
+  .canopy-list {
+    font-weight: 400;
+    font-size: 18px;
+  }
+
+  .Tonic-content {
+    font-size: 16px;
+  }
+
+  .Ruby {
+    display: block;
+  }
+
+  .disable {
+    color: #6070ff;
+    border: 1px solid #7f8cff;
+    pointer-events: fill;
+    cursor: default;
+  }
+
+  .disable:hover {
+    background-color: #6070ff;
+    color: #fff;
+    text-decoration: underline;
+  }
+
+  .disable:active {
+    background: #2230d2;
+    color: #fff;
+  }
+
+  .action-button {
+    display: flex;
+  }
+
+  .three {
+    order: 1;
+  }
+
+  .wrap-footer {
+    display: flex;
+  }
+
+  .wrap-footer-one {
+    margin-left: 4rem;
+    margin-right: 0;
+    flex-basis: 500px;
+  }
+
+  .flex-connect-three {
+    font-size: 16px;
+    font-weight: 500;
+    color: #7f8cff;
+  }
+
+  .flex-connect-three h3 {
+    margin-left: 0;
+  }
+
+  .wrap-footer-two {
+    flex-basis: 700px;
+    margin-left: 2rem;
+  }
+
+  .language ul {
+    margin-top: 7rem;
+    justify-content: space-between;
+    margin-right: 6rem;
+  }
+
+  .language p {
+    font-size: 30px;
+    font-weight: 600;
+  }
+
+  .lang-wrapper {
+    margin-left: 2rem;
+  }
+
+  .lang2 {
+    display: inline-block;
+    margin: 0 5px;
+    padding-left: 3px;
+    padding-right: 30px;
+  }
+
+  .Frameworks1 ul {
+    margin-right: 6rem;
+  }
+
+  .Skills ul {
+    margin-right: 6rem;
+  }
+
+  footer {
+    margin: 0;
+  }
+
+  .resume {
+    margin: 15px;
+  }
+
+  .lang2 ul {
+    display: block;
+  }
+
+  .contact-details {
+    background-image: url(./desktop-contactimage.png);
+    display: flex;
+    justify-content: center;
+    padding-bottom: 4rem;
+  }
+
+  form {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .contact-content {
+    padding-bottom: 10px;
+  }
+
+  .contact-content p {
+    font-size: 20px;
+    font-weight: 300;
+    text-align: center;
+  }
+
+  input {
+    padding-right: 20rem;
+    border-radius: 0;
+  }
+
+  textarea {
+    border-radius: 0;
+    padding-right: 20rem;
+  }
```

Comment:
- [ ] Kindly make sure to follow the Figma design guidelines and make the input field of all to equal.
- [ ] Kindly make sure that paragraph is of the correct width.
- Your design:
<img width="999" alt="Screenshot 2022-03-23 at 9 14 31 PM" src="https://user-images.githubusercontent.com/70269507/159769228-44def815-5aad-4a08-92b4-5f067ea3a0fe.png">

- Figma design:
<img width="457" alt="Screenshot 2022-03-23 at 9 17 28 PM" src="https://user-images.githubusercontent.com/70269507/159769306-b2654996-54e7-40c5-85d5-08ea1d8b2f4c.png">

Reply:
@BenMukebo Please run this link in your browser to confirm I have the right changes. 
https://raw.githack.com/kwambiee/First-Portfolio/desktop-version/index.html